frontend/projects/upgrade/src/app/features/dashboard/experiments/modals/edit-condition-weights-modal/edit-condition-weights-modal.component.html

```diff
@@ -0,0 +1,122 @@
+<app-common-dialog
+  [title]="config.title"
+  [cancelBtnLabel]="config.cancelBtnLabel"
+  [primaryActionBtnLabel]="config.primaryActionBtnLabel"
+  [primaryActionBtnColor]="config.primaryActionBtnColor"
+  [primaryActionBtnDisabled]="isPrimaryButtonDisabled$ | async"
+  (primaryActionBtnClicked)="onPrimaryActionBtnClicked()"
+>
+  <form [formGroup]="conditionWeightForm" class="form-standard dense-3">
+    <div class="weight-method-section dense-1">
+      <mat-radio-group formControlName="weightingMethod">
+        <span class="section-label ft-14-600">{{ 'experiments.edit-condition-weights-modal.method-header.text' | translate }}</span>
+        <mat-radio-button
+          *ngFor="let method of weightingMethods"
+          [value]="method.value"
+          [disabled]="method.disabled"
+        >
+          <div class="radio-content">
+            <span
+              class="radio-label ft-14-600"
+              [class.disabled-text]="method.disabled"
+            >
+              {{ method.name | titlecase }}
+            </span>
+            <span
+              class="radio-description ft-12-400"
+              [class.disabled-text]="method.disabled"
+            >
+              {{ method.description | translate }}
+            </span>
+          </div>
+        </mat-radio-button>
+      </mat-radio-group>
+    </div>
+
+    <div class="table-container">
+   <table
+    mat-table
+    [dataSource]="conditions"
+    class="conditions-table"
+    [ngClass]="{ 'no-data': !conditions?.length }"
+  >
+    <!-- Condition Column -->
+    <ng-container matColumnDef="condition">
+      <th mat-header-cell *matHeaderCellDef class="condition-column ft-14-600">
+        Condition
+      </th>
+      <td mat-cell *matCellDef="let condition" class="condition-column ft-14-400">
+        {{ condition.conditionCode }}
+      </td>
+      <!-- Summary row for condition column -->
+      <td mat-footer-cell *matFooterCellDef class="condition-column ft-14-400 summary-row">
+        @if (getTotalWeightStatus()?.totalWeightInvalid) {
```

Comment:
this would be the only place we're using the newer angular @if syntax over `ngIf`. i'm in favor of this syntax generally but just noting it's different, we haven't really talked about using the newer syntax features.

Reply:
i think migrating to the new syntax is something we should commit to doing when we do the next angular update, i see no harm in that, seems better, more like react/jsx. so if you want to leave this as is, the rest of the codebase can catch up later.